src/webui/api/synccontroller.cpp

```diff
@@ -482,3 +507,19 @@ void SyncController::torrentPeersAction()
     sessionManager()->session()->setData(QLatin1String("syncTorrentPeersLastResponse"), lastResponse);
     sessionManager()->session()->setData(QLatin1String("syncTorrentPeersLastAcceptedResponse"), lastAcceptedResponse);
 }
+
+qint64 SyncController::getFreeDiskSpace()
+{
+    const qint64 now = QDateTime::currentMSecsSinceEpoch();
+    const auto thirtySeconds = 30000;
```

Comment:
When I ask you to create named constant for it I meant something other and I hoped it's so obvious to understand it without any additional description.
Usually we create such "compile time configuration" constants at top of the file for easily changing it in the future without searching through all the code. And it should have an appropriate name to easily understand what is it configure (e.g. FREEDISKSPACE_CHECK_TIMEOUT).
It's absolutely pointless to have it at this place.

Reply:
> It's absolutely pointless to have it at this place.

Not totally pointless, it clearly documents the length of time between calculation. I will move it to the top of the file.

Reply:
Moved this constant to the top of `SyncController` and squashed